src/varlink.js

```diff
@@ -0,0 +1,97 @@
+
+import cockpit from "cockpit";
+
+const encoder = cockpit.utf8_encoder();
+const decoder = cockpit.utf8_decoder();
+
+/*
+ * Connect to a varlink service.
+ *
+ * Returns a connection object:
+ *
+ * - connection.call(method, parameters) calls "method" (prefixed by the
+ *   interface) with "parameters". Multiple calls can be made without waiting
+ *   for replies, but calls are made sequentially (similar to http). Returns a
```

Comment:
https://varlink.org/ "Protocol" says the opposite, or it is misleading. At least when I tried to do multiple parallel varlink calls to podman,  things went horribly wrong.

Reply:
Requests can be pipelined, but they will still responded to in order. I just verified again that podman does this correctly.

Does changing the wording on varlink.org to this help clarify: https://github.com/varlink/varlink.github.io/pull/13 ?

Reply:
A bit. TBH I'm surprised that this is even a legal mode of operation, as it makes state keeping dramatically harder on both sides, and it's not well defined any more when multiple responses to a call come into play. But, this isn't the place to discuss the protocol. :-)